packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2155,6 +2158,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         ackHandle: string,
         trackerRefSeqNum: number, // back-compat summarizerNode - remove when fully enabled
         version?: IVersion,
+        summaryLogger: ITelemetryLogger = this.logger,
```

Comment:
If there are not that many callers, I'd rather make it non-optional and make everybody pass it properly